README.md

```diff
@@ -31,17 +31,18 @@ Or install it yourself as:
 ### Pipeline to ingest organizations into Rialto
 
 ```
-exe/extract call StanfordOrganizations > step1.json
-exe/transform call StanfordOrganizationsToJsonList -i step1.json > step2.json
-exe/transform call OrganizationsListToJSONLD -i step2.json > step3.jsonld
-SETTINGS__TOKENS__RIALTO=<Rialto API key> exe/load call Sparql -i step3.jsonld
+exe/extract call StanfordOrganizations > organizations.json
+exe/transform call StanfordOrganizations -i organizations.json > organizations.sparql
+SETTINGS__TOKENS__RIALTO=<Rialto API key> exe/load call Sparql -i organizations.sparql
```

Comment:
How would you feel about removing `SETTINGS__TOKENS__RIALTO=<Rialto API key> ` from this line, since that is covered elsewhere in the README?

Reply:
(I'd also be happy to do this in a follow-up PR, since you didn't add this here.)

Reply:
Yeah, I think a followup.  There's no need for Justin to fix Justin's omissions 😆